beaker-vue/src/components/panels/FilePanel.vue

```diff
@@ -223,12 +212,15 @@ const doubleClick = ({data}) => {
     }
   }
   else {
-    showToast({
-      title: 'Unable to handle file',
-      detail: 'Unable to handle this file. Please choose a different file.',
-      severity: 'warning',
-      life: 2500,
-    })
+    if (data.type !== 'directory') {
+      if (data.size >= largeFileWarningSize) {
+        largeFileSizeUILabel.value = `approximately ${Math.floor(data.size / 1000000)} MB`;
+        showLargeFilePreview(() =>
+          previewFile(data.path, data.mimetype));
```

Comment:
If you could open/switch to the correct panel when this is run, that would be awesome.